internal/util/apiclient/websocket_log_reader.go

```diff
@@ -59,33 +59,28 @@ func ReadWorkspaceLogs(activeProfile config.Profile, workspaceId string, project
 			continue
 		}
 
-		readJSONLog(ws, stopLogs, logs_view.WORKSPACE_INDEX)
+		ReadJSONLog(ws, logs_view.WORKSPACE_INDEX, stopLogs)
 		ws.Close()
 		break
 	}
 
 	wg.Wait()
 }
 
-func readJSONLog(ws *websocket.Conn, stopLogs *bool, index int) {
+func ReadJSONLog(ws *websocket.Conn, index int, stopLogs ...*bool) {
```

Comment:
Why this change?

I don't understand why the `stopLogs` argument would be a list of bool pointers.

If you were aiming for an "optional" param. Just sending `false` is enough I think.